src/cmd_vel_mux.cpp

```diff
@@ -135,15 +136,14 @@ bool CmdVelMux::parametersAreValid(const std::map<std::string, ParameterValues>
 
 void CmdVelMux::configureFromParameters(const std::map<std::string, ParameterValues> & parameters)
 {
-  std::map<std::string, ParameterValues>::const_iterator parameter;
   std::map<std::string, std::shared_ptr<CmdVelSub>> new_map;
 
   for (const std::pair<std::string, ParameterValues> & parameter : parameters)
   {
-    std::string key = parameter.first;
-    ParameterValues parameter_values = parameter.second;
+    const std::string &key = parameter.first;
```

Comment:
@Lobotuerk nit:

```suggestion
    const std::string & key = parameter.first;
```

Same elsewhere.